lib/pagination/adapter.js

```diff
@@ -32,8 +43,8 @@ const contentPagination = adapterParams => ({
   previous_page: previousPage(adapterParams),
   current_page: adapterParams.page,
   next_page: nextPage(adapterParams),
-  previous_page_url: pageUrl(adapterParams, previousPage(adapterParams)),
-  next_page_url: pageUrl(adapterParams, nextPage(adapterParams))
+  previous_page_link: pageUrl(adapterParams, previousPage(adapterParams)),
```

Comment:
is there a way we are changing the word previous_page_url for previous_page_link ? This is a breaking change with previous versions

Reply:
I read that this change was due to standards/convention. Is there a way to let the mantainers know that they should bump the mayor version number ?

Reply:
I guess not. I am the only maintainer right now as far as I know, and as this package is still gaining strength maybe it lack of many things. What do you suggest to do to bump versions properly?

Reply:
That depends of how you manage the versioning. I am guessing you would like to use something like https://semver.org/ .In that case the change you are introducing is a breaking change because if someone was using the `previous_page_url` or `next_page_url` attributes they will no longer find it and the app is going to crash so you will need to bump you MAYOR versión. On the other hand if you decide to just push the fix and maintain the original attributes, I would bump the PATCH version.
Since MAYOR versions are not retrocompatible, most persons don't have automatic updates that can jump between mayor versions so they wont receive the fix.

Reply:
Thanks for the explanation! I agree that major change would force a bump to major version. I think it's not a problem, since there is only an internal project using this package. But I will discuss if there's no problem in bumping a major version, or I just let the real fix to the bug for a minor version and let the changed params for another bump. Thanks again!